### ISSUE TITLE
Extremely preterm

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7269]
**Preferred term label:**
Extremely preterm

**Synonyms**

**Definition (free text, please give PubMed ID)**
Extremely preterm is defined as babies born alive before 28 weeks of gestation (PMID: 27743648). 

**Parent term (use hpo.jax.org/app)**
Premature birth

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
added, thanks
